README.md

```diff
@@ -79,21 +79,17 @@ Now build OpenWrt in the standard way:
 
     $ make V=s -j1
 
-Once the build is completed, you will find the resulting output i.e. images, dtbs and rootfs at "bin/pistachio", depending upon the selected profile.
-Where PROFILE can be marduk_cc2520, marduk_ca8210, marduk_cc2520_wifi and VERSION is whatever mentioned in CONFIG_VERSION_NUMBER.
-By default VERSION is blank if you do not use the creator-platform-default.config for loading the configuration.
+Once the build is completed, you will find the resulting output i.e. images, dtbs and rootfs at "bin/pistachio".
 
-- openwrt-$(VERSION)-pistachio-pistachio_$(PROFILE)-uImage
-- openwrt-$(VERSION)-pistachio-pistachio_$(PROFILE)-uImage-initramfs
-- openwrt-$(VERSION)-pistachio-marduk-$(PROFILE)-rootfs.tar.gz
-- pistachio_marduk_ca8210.dtb (for marduk_ca8210 board)
+Example output for V1.0.0:
```

Comment:
If we are making 0.9.2 for production then should we just that OK instead of imaginary 1.0.0?